setup.py

```diff
@@ -19,7 +19,6 @@
     zip_safe=True,
     install_requires=[
         'setuptools',
-        'opencv-python',
```

Comment:
Will this still be installed on OSX systems if it only appears in the package.xml ? (i genuinely don't know)

Reply:
I don't really have an OSX to test it, but it depends on rosdep database. I've been looking for some information and I've found this:

https://github.com/ros/rosdistro/blob/ed224748780c22f029767f481af24889aff08257/rosdep/python.yaml#L6067-L6078

They've specified `python3-opencv` for several platforms but not OSX. But there's an specific database file for it and in there I've also found this:

https://github.com/ros/rosdistro/blob/ed224748780c22f029767f481af24889aff08257/rosdep/osx-homebrew.yaml#L554-L557

So, for OSX do exists `python-opencv`. To make `package.xml` more generalist, I've added conditions for different python versions which ROS can use:

https://github.com/icaropires/ros2_video_streamer/blob/dd9cefa678bb1314983bc74062a2015d0c192852/package.xml#L16-L17

Additionally, it seems that this is how it's done in general. Some `package.xml` references:
* [vision_opencv](https://github.com/ros-perception/vision_opencv/blob/d9e88980feaf997a282da5acd0da2a0ed0583ea4/cv_bridge/package.xml#L25)
* [detection_visualizer](https://github.com/ros2/detection_visualizer/blob/4f39e9dd1f3b2727a19e1270a69accbd78b67c54/package.xml#L14)
* [slide_show](https://github.com/ros2/slide_show/blob/4df57c2e877e4ea2c8d025d9168a089e6ef1d548/package.xml#L13)
* [jsk_topic_tools](https://github.com/jsk-ros-pkg/jsk_common/blob/ef394804ecd052fc0620918d1c4ad91d98c3771a/jsk_topic_tools/package.xml#L49-L50)